src/components/EntityMap.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <MglMap
+    :zoom="10"
+    :accessToken="accessToken"
+    :mapStyle="mapStyle"
+    :center="center"
+    @load="onMapLoaded"
+  >
+    <MglMarker
+      v-for="(coordinates, index) in coordinatesList"
+      :key="`${coordinates[2]}-${coordinates[3]}-${index}`"
+      :coordinates="[coordinates[0], coordinates[1]]"
+      @click="onMarkerClick(coordinates)"
+    >
+      <MglPopup
+        v-if="coordinates[3]"
+        :closeButton="false"
+        :closeOnClick="true"
+        :showed="selectedPlace === coordinates[2]"
```

Comment:
@paltman I'm seeing some weird sync issues with how `showed`, the click handler, and the `onSelect` method are interacting:

https://share.getcloudapp.com/E0uzJoX9



![image](https://user-images.githubusercontent.com/629062/86974343-c759be00-c13b-11ea-9383-914592060079.png)

It looks like `showed` is "non-synced" (I don't know if that is a Vue thing or a vue-mapbox thing...maybe if we include the value of `selectedPlace` in the `MglMarker` key that would effectively resolve the sync issue.
https://soal.github.io/vue-mapbox/api/popup.html#props

Reply:
@jacobwegner does 8467d9f help?  I'm not sure how to repro what that screenshot is showing.

Reply:
@paltman The synchronization is working better, but now the popup is not triggered when clicking a map marker:

https://share.getcloudapp.com/o0uv0QmE


I think we want all of these things to work consistently

-  A click on a map marker to select the entity (highlight color, sidebar selection, popup shown)
- Clearing the selection from the sidebar widget clears everywhere (highlight color, sidebar selection, popup shown)
- Clicking another entity (be it in the sidebar, in the text, or a map marker) changes the entity (highlight color, sidebar selection, popup shown)

If we can ensure the popup is shown when selecting the map marker, then I think we've implemented the feature fully.


I've moved ![](https://github.trello.services/images/mini-trello-icon.png)  [Synchronize named entity selection with the map](https://trello.com/c/zbYYPfwq) back to "Pull Request" and blocked until this is resolved.

Reply:
@paltman use this path to debug:

`/reader?urn=urn%3Acts%3AgreekLit%3Atlg0012.tlg001.perseus-eng4%3A2.480`

Reply:
@paltman and I decided to defer sync issues until after this branch was merged (so we could pick up on all the other great refactoring).